code/_onclick/hud/hud.dm

```diff
@@ -105,8 +105,8 @@
 	return ..()
 
 /mob/proc/create_mob_hud()
-	return
-
+	if(client && !hud_used)
+		hud_used = new /datum/hud(src)

```

Comment:
because I didn't realise that all mobs need a /datum/hud , even if it's pretty much empty.